index.html

```diff
@@ -263,7 +264,13 @@ <h3>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
-            <li>An API for Verifiable Credential Exchange</li>
+            <li>A Developer Guide consisting of one or more notes related to:
+              <ul>
+                <li>General implementaion guidance and best practices for working with VCs</li>
+                <li>A REST API definition for Verifiable Credential Exchange</li>
```

Comment:
```suggestion
                <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
```

Some people go apoplectic when they hear the word "REST". I prefer REST, just trying to generalize the thing we're talking about so the anti-REST crowd doesn't show up.

Reply:
I am good with this.